README.md

```diff
@@ -199,13 +199,25 @@ This method will call the terminate method on each worker.
 
 Performance is one of the main target of these worker pool implementations, we want to have a strong focus on this.  
 We already have a bench folder where you can find some comparisons.  
-**Thread pools** ( FixedThreadPool and DynamicThreadPool ) are suggested to run CPU intensive tasks, you can still run I/O intensive tasks into thread pools, but performance enhancement is expected to be minimal.  
+
+Before to jump into each poolifier pool type, let highlight that **Node.js comes with a thread pool already**, the libuv thread pool where some particular tasks already run by default.  
+Please take a look at [which tasks run on the libuv thread pool](https://nodejs.org/en/docs/guides/dont-block-the-event-loop/#what-code-runs-on-the-worker-pool).  
+
+Now **if your task runs on libuv thread pool**, you can :
+
+ -Increase the libuv thread pool size setting the [UV_THREADPOOL_SIZE](https://nodejs.org/api/cli.html#cli_uv_threadpool_size_size) and/or  
+ -Use poolifier cluster pool that spawning child processes will also increase the number of libuv threads since that any new child process comes with a separated libuv thread pool.
```

Comment:
I need to be 100% sure ot this statement

Reply:
Ok I did another check monitoring the number of total threads with top command and using a pool with 50 processes.
So when I start the process I see much more threads ( aroung 300 more ), when I stop it the number decrease